applications/IgaApplication/python_scripts/iga_analysis.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics
+import KratosMultiphysics.IgaApplication as IgaApplication
+
+# Importing the solvers (if available)
+try:
+    import KratosMultiphysics.ExternalSolversApplication
+    KratosMultiphysics.Logger.PrintInfo("ExternalSolversApplication", "succesfully imported")
+except ImportError:
+    KratosMultiphysics.Logger.PrintInfo("ExternalSolversApplication", "not imported")
+try:
+    import KratosMultiphysics.EigenSolversApplication
+    KratosMultiphysics.Logger.PrintInfo("EigenSolversApplication", "succesfully imported")
+except ImportError:
+    KratosMultiphysics.Logger.PrintInfo("EigenSolversApplication", "not imported")
+
+
+# Importing the base class
+from analysis_stage import AnalysisStage
+
+class IgaAnalysis(AnalysisStage):
+    """
+    This class is the main-script of the IgaApplication put in a class
+
+    It can be imported and used as "black-box"
+    """
+    def __init__(self, model, project_parameters):
```

Comment:
you can remove this constructor. I don't think you will use MPI anytime soon ...